Panic: nil pointer dereference when creating VPS - `paid_till` is null during initialization

### Versions
Terraform v1.5.7
terraform-provider-ruvds v1.3.2

### Affected Resource(s)
- `ruvds_vps`

### Terraform Configuration

```hcl
resource "ruvds_vps" "ruproxy" {
  datacenter_id   = 5      # M9 Moscow
  tariff_id       = 14     # Regular
  payment_period  = 2      # 1 month
  os_id           = 255    # Ubuntu 24.04 LTS
  cpu             = 1
  ram             = 1
  drive           = 20
  drive_tariff_id = 1      # HDD
  ip              = 1
  computer_name   = "ruproxy"
  user_comment    = "HAProxy TCP proxy"
  ssh_key_id      = ruvds_ssh.deploy_key.ssh_key_id
}
```

### Debug Output

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x305fdc7]

goroutine 57 [running]:
github.com/rustamkulenov/terraform-provider-ruvds/internal/provider/resources.(*VpsResource).Create(0xc00033b470, {0x3407018, 0xc000352630}, {{{{0x340bf10, 0xc0003662a0}, {0x32f5560, 0xc000353d70}}, {0x340da58, 0xc0003345f0}}, {{{0x340bf10, ...}, ...}, ...}, ...}, ...)
	github.com/rustamkulenov/terraform-provider-ruvds/internal/provider/resources/vps_resource.go:246 +0x4c7
```

### Expected Behavior

The VPS should be created successfully and Terraform should handle the case where `paid_till` is null during the "initializing" phase.

### Actual Behavior

The provider crashes with a nil pointer dereference when trying to access `response.Status.PaidTill`.

### Root Cause Analysis

When the RUVDS API creates a new VPS, it returns the following response structure:

```json
{
  "virtual_server_id": 2403532,
  "status": {
    "virtual_server_id": 2403532,
    "status": "initializing",
    "create_progress": 0,
    "paid_till": null,
    "network_v4": null
  }
}
```

The `paid_till` field is `null` because the server is still initializing. However, in `vps_resource.go` line 247, the code dereferences this null pointer without checking:

```go
plan.PaidTill = types.StringValue(*response.Status.PaidTill)
```

### Proposed Fix

Add nil checks before dereferencing pointer fields:

```go
// Save received values in state
plan.ID = types.Int32Value(response.VirtualServerId)
if response.Status.Status != nil {
    plan.Status = types.StringValue(*response.Status.Status)
}
plan.CreateProgress = types.Int32Value(response.Status.CreateProgress)
if response.Status.PaidTill != nil {
    plan.PaidTill = types.StringValue(*response.Status.PaidTill)
}
if len(response.Status.NetworkV4) > 0 {
    plan.IPAddress = types.StringValue(response.Status.NetworkV4[0].IPAddress)
    plan.Netmask = types.StringValue(response.Status.NetworkV4[0].Netmask)
    plan.Gateway = types.StringValue(response.Status.NetworkV4[0].Gateway)
}
```

### Steps to Reproduce

1. Configure the provider with a valid API token
2. Create an `ruvds_ssh` resource
3. Create an `ruvds_vps` resource with valid configuration
4. Run `terraform apply`
5. The provider crashes during VPS creation

### Additional Context

The API call itself succeeds (tested with curl), but the provider crashes while processing the response. The same issue may affect other pointer fields in the `VirtualServer` struct.

### References

- VPS API documentation: https://ruvds.com/en-usd/help/api
- Similar issue pattern: #16 (Bug?)

--- a/internal/provider/resources/vps_resource.go
+++ b/internal/provider/resources/vps_resource.go
@@ -243,7 +243,9 @@ func (r *VpsResource) Create(ctx context.Context, req resource.CreateRequest, re
 	plan.ID = types.Int32Value(response.VirtualServerId)
 	plan.Status = types.StringValue(*response.Status.Status)
 	plan.CreateProgress = types.Int32Value(response.Status.CreateProgress)
-	plan.PaidTill = types.StringValue(*response.Status.PaidTill)
+	if response.Status.PaidTill != nil {
+    	plan.PaidTill = types.StringValue(*response.Status.PaidTill)
+	}
 	if len(response.Status.NetworkV4) > 0 {
 		plan.IPAddress = types.StringValue(response.Status.NetworkV4[0].IPAddress)
 		plan.Netmask = types.StringValue(response.Status.NetworkV4[0].Netmask)

